src/pods/canvas/components/relation/database-relation-collection.component.tsx

```diff
@@ -62,7 +81,7 @@ export const DatabaseRelationCollectionComponent: React.FC<
             isSelected={relation.id === schema.selectedElementId}
           />
         ) : (
-          <DatabaseRelationSelfComponent
+          <DatabaseRelationshipTwoTablesStraightComponent
             id={relation.id}
             onClick={onSelectRelation}
             onDoubleClick={onEditRelation}
```

Comment:
Create a type

type layoutType = "self" | "straight" | "overlapping";

And in business create a method

getCalculationTypeOfLayout = (...whatever is needed) : LayoutType


the use a funtction and use a switch case for this (easier to read)

e.g.

```tsx
function Notification({ text, status }) {
  switch (status) {
    case 'info':
      return <Info text={text} />;
    case 'warning':
      return <Warning text={text} />;
    case 'error':
      return <Error text={text} />;
    default:
      return null;
  }
}
```

Read this post: https://www.robinwieruch.de/conditional-rendering-react/